<launch>
    <!-- 摄像头节点 -->
    <node name="head_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <!--这是手臂摄像头的launch文件，请修改-->
        <param name="video_device" value="/dev/video0" />
   	    <param name="image_width" value="640" />
    	<param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="head_camera_link" />
        <param name="io_method" value="mmap"/>
        <param name="camera_name" value="head_camera"/>
        <!--param name="camera_info_url" type="string" value="file://$(find ar_pose)/cam_info/head_camera.yaml"/-->
    </node>
    <node name="face_rec_topic" pkg="face_rec" type="face_rec_topic.py" output="screen">
        <!-- 人脸数据库目录 -->
        <param name="face_data" value="$(find face_rec)/face_data/"/>
        <!--识别容差-->
        <param name="tolerance" value="0.5"/>
    </node>
</launch>
